x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/delete_domain_panel.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import { EuiButton, EuiSpacer, EuiText } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { CrawlerSingleDomainLogic } from '../crawler_single_domain_logic';
+
+export const DeleteDomainPanel: React.FC = ({}) => {
+  const { domain } = useValues(CrawlerSingleDomainLogic);
+  const { deleteDomain } = useActions(CrawlerSingleDomainLogic);
+
+  if (!domain) {
+    return null;
+  }
+
+  return (
+    <>
+      <EuiText size="s">
+        <p>
+          <FormattedMessage
+            id="xpack.enterpriseSearch.appSearch.crawler.deleteDomainPanel.description"
+            defaultMessage="Remove this domain from your crawler. This will also delete all entry points and crawl
+                    rules you have setup. {cannotUndoMessage}."
+            values={{
+              cannotUndoMessage: (
+                <strong>
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.appSearch.crawler.deleteDomainPanel.cannotUndoMessage',
+                    {
+                      defaultMessage: 'This cannot be undone',
+                    }
+                  )}
+                </strong>
+              ),
+            }}
+          />
```

Comment:
Not sure I understand - what would that look like?

Reply:
```
{i18n.translate({defaultMessage: 'Remove this domain from your crawler. This will also delete all entry points and crawl rules you have setup.',})}{' '}
<strong>{i18n.translate({defaultMessage: 'This cannot be undone.',})}</strong>
```

It's kind of pedantic.  We have to work around the `<strong></strong>` tags so its a decision of whether the sentence is "nested" in the first or next to the second.  I don't think we've established a standard around this pattern.

Reply:
Seems more readable in the current form, so I would probably keep as is.